src/renderer/src/context/game-details/game-details.context.tsx

```diff
@@ -141,7 +141,7 @@ export function GameDetailsContextProvider({
         filters: [
           {
             name: "Game executable",
-            extensions: ["exe"],
+            extensions: ["exe", "bash", "sh", "bat"], //TODO: check if empty extension can be added, so it's possible to select linux binaries without extinsion. Add if can be added.
```

Comment:
AFAIK empty is not a valid option but could have something like this.

```typescript
{ name: 'All Files', extensions: ['*'] }
```

Reply:
It would be so wonderful if it was possible to filter files by their execution permissions. I think this solution is good enough though